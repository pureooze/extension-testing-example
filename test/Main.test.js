import { h } from "preact";
import { render, cleanup } from "preact-testing-library";
import Main from "@/example/index";
import chrome from "sinon-chrome"; // from 'sinon-chrome'

window.chrome = chrome;

test("should render Main component", () => {
  const { container } = render(<Main />);
  expect(container).toMatchSnapshot();
});
