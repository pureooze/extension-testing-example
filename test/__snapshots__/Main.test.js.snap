// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Main component 1`] = `
<div>
  <div>
    <button
      id="changeColor"
    />
    <b>
      
    </b>
    ;
  </div>
</div>
`;
